packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionTncSection/JurisdictionTncSection.tsx

```diff
@@ -34,7 +34,7 @@ const JurisdictionTncSection = ({
     const selectedCompany = companyNamesAndUrls[selectedJurisdiction as keyof typeof companyNamesAndUrls];
 
     return (
-        <div className='text-center space-y-12 mt-30 sticky bottom-0 bg-system-light-primary-background px-20 pt-[15px] pb-10 w-screen sm:w-auto sm:bg-inherit sm:static sm:p-0'>
+        <div className='text-center space-y-12 mt-30 sticky bottom-0 bg-system-light-primary-background px-20 pt-[15px] pb-10 sm:w-auto sm:bg-inherit sm:static sm:p-0'>
```

Comment:
I believe here we dont need to specify sm, cause then this wont work for tablet. We just need to specify for lg cases. Everything else is default

Reply:
I just pushed some fixes which needs sm for the mobile view to work properly